templates/accounts/login_signup.html

```diff
@@ -69,7 +70,17 @@ <h1>Sign in</h1>
             <input type="email" name="email" placeholder="Email" value="{{ enered_email }}"
                 pattern="[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?"
                 required />
-            <input type="password" name="password" placeholder="Password" required />
+
+            <!-- Div to hold the elements in single line -->
+            <div style="display: table;width: 100%;margin-left: -12px;">
+
+                <input type="password" name="password" id='password' style="display: inline-block" placeholder="Password" required>
+
+                <!-- Eye icon to toggle -->
+                <i class="far fa-eye" id="togglePassword" style="margin-left: -30px; cursor: pointer;"></i>
```

Comment:
Could you add some sign or something on hover which will inform the user that this icon is to show/hide the password.

Reply:
A tooltip should be fine.